components/test_raftstore/src/pd.rs

```diff
@@ -311,6 +315,20 @@ impl Cluster {
         }
     }
 
+    fn get_all_stores(&self) -> Result<Vec<metapb::Store>> {
+        Ok(self
```

Comment:
Why filter stores with id == 0? Is it possible?

Reply:
Yes. It's zero means it's a placeholder during testing cluster initializing:

https://github.com/tikv/tikv/blob/90d0739beb6b2213bb2e3db314226b080995e019/components/test_raftstore/src/pd.rs#L288-L305